order/manager.go

```diff
@@ -36,13 +35,6 @@ var (
 	// implement the same batch verification version as the server.
 	ErrVersionMismatch = fmt.Errorf("version %d mismatches server version",
 		CurrentBatchVersion)
-
-	// nodeIdentityKeyLoc is the key locator from which the identity key of
-	// an lnd node is derived.
-	nodeIdentityKeyLoc = keychain.KeyLocator{
```

Comment:
any reason we opted for the node key to begin with?

Reply:
Not sure, one other added benefit of this change is that a ticket is no longer bound to a particular _node_, but rather an account in the system.